app/models/rbac.rb

```diff
@@ -222,7 +222,7 @@ def self.find_targets_filtered_by_parent_ids(parent_class, scope, find_options,
   end
 
   def self.find_targets_filtered_by_ids(scope, find_options, u_filtered_ids, filtered_ids)
-    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count
+    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count(:all)
```

Comment:
It looks like a decision of when to use count(:all) vs. count was made in cf583b0456f1448e403280886ca3e870ea7bc167.  I don't follow the difference.  Maybe @kbrock remembers.